### PR TITLE
added wishlist page

### PR DESCRIPTION
**Pull request by tusharkandpal**
- wishlist html consists of main, section & cards
- added wishlist styles

NOTE: will create color variables atlast

### CODE REVIEW

File: components/wishlist-page/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+/* ------------- || COMMON CSS ------------------ */
+.wishlist-main,
+.wishlist-section {
+  padding: 2rem 0;
+}
+
+/* ------------------------ || WISHLIST MAIN -------------------------- */
+.wishlist-main {
+  background-color: #f7f7f7;
+}
```
Comment (sushil-mod) [inline components/wishlist-page/wishlist.css:13]: if possible use color variable

**Review by sushil-mod — APPROVED** [review]
after gone through comment , you can merge!

File: components/wishlist-page/wishlist.css (hunk 1 above)
Comment (tusharkandpal) [reply components/wishlist-page/wishlist.css:13]: will create all variables at last

**Review by shaksham08 — APPROVED** [review]
Looks good

**Pull request merged**